test/parallel/test-fs-write-sync-without-offset-and-length.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
Okay, I hear you. Fixed in 15d9002.

Did the same thing for the `fs.write` tests in 628ee89d. Let me know if you'd like that included as well.